src/wasm.rs

```diff
@@ -126,13 +126,29 @@ pub fn now() -> f64 {
 
 // The JS now function is in a module so it won't have to be renamed
 #[cfg(not(any(feature = "wasm-bindgen", feature = "stdweb")))]
+#[cfg(not(target_os = "emscripten"))]
 mod js {
     extern "C" {
         pub fn now() -> f64;
     }
 }
 // Make the unsafe extern function "safe" so it can be called like the other 'now' functions
 #[cfg(not(any(feature = "wasm-bindgen", feature = "stdweb")))]
+#[cfg(not(target_os = "emscripten"))]
 pub fn now() -> f64 {
     unsafe { js::now() }
 }
+
+#[cfg(not(any(feature = "wasm-bindgen", feature = "stdweb")))]
+#[cfg(target_os = "emscripten")]
+mod js {
+    extern "C" {
+        pub fn _emscripten_get_now() -> f64;
+    }
+}
+// Make the unsafe extern function "safe" so it can be called like the other 'now' functions
+#[cfg(not(any(feature = "wasm-bindgen", feature = "stdweb")))]
+#[cfg(target_os = "emscripten")]
+pub fn now() -> f64 {
+    unsafe { js::_emscripten_get_now() }
+}
```

Comment:
I think this can be more consise:

```rust
mod js {
    extern "C" {
       #[cfg(not(target_os = "emscripten"))]
        pub fn now() -> f64;
        #[cfg(target_os = "emscripten")]
        pub fn _emscripten_get_now() -> f64;
    }
}
// Make the unsafe extern function "safe" so it can be called like the other 'now' functions
#[cfg(not(any(feature = "wasm-bindgen", feature = "stdweb")))]
pub fn now() -> f64 {
    #[cfg(not(target_os = "emscripten"))]
    return unsafe { js::now() };
    #[cfg(target_os = "emscripten")]
    return unsafe { js::_emscripten_get_now() };
}
```

Reply:
will do

Reply:
done 👍